examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL/SDL.h>
```

Comment:
also you can use the power of docker: https://github.com/hunter-packages/travis-linux-docker